pages/ugm2023.md

```diff
@@ -70,6 +70,25 @@ There are many other hotel and guest house options accesible to and from the ven
 
 Citizens of several countries may travel to Portugal to attend a meeting without obtaining a visa. Please visit the [Portugal Ministry of Foreign Affairs website][mne.gov.pt] for more information about travel visas. Confirmed attendees requiring a letter clarifying the purpose of travel should [contact the MDAnalysis project manager][email]. 
 
+## Health and Safety
+We are committed to providing a safe, comfortable, and healthy environment for the MDAnalysis UGM and have looked to local and international entities and other organziations in the open source community to develop onsite safety measures for UGM attendees.
+
+### Vaccine Policy
+MDAnalysis strongly recommends that UGM attendees remain current (i.e., receiving all recommended primary and booster doses recommended by your local government) with their COVID-19 vaccinations as a personal protective measure. 
+
+### Testing
+A COVID-19 test result is not required to determine whether you can attend the UGM. Rapid self-tests will be made available at check-in for those who would like to test themselves for COVID-19 during the meeting.
+
+*Testing Positive for COVID-19*
+
+Individuals who are COVID-positive should not attend the UGM. If you receive a positive COVID-19 test result during the UGM, please notify members of the MDAnalysis UGM organizing committee and do not return to the event unless you have been cleared by the committee.
+
+### Face Coverings
+Face coverings are encouraged, but not required, in indoor spaces during the UGM. We view face coverings as a personal choice for protecting oneself and will support attendees in their choice to wear or not to wear a face covering.
+
+### Social Distancing and Hygiene
+While we will be unable to enforce social distancing guidelines during the UGM, there is extra seating capacity at the venue that allows for voluntary distancing. At check-in, you will have the option to indicate your comfort level (e.g., handshakes/high-fives are OK, elbow bump greetings preferred, no-contact greetings and more personal space desired) on your name tag.
```

Comment:
Do we need to get some tiny dot stickers for this?

Reply:
Yesterday I bought a bunch of different colored dot stickers that we can use for this and for indicating whether people would like to be asked before being in a photo.